workflows/multiomics/integration/scanorama_leiden/nextflow_params.yaml

```diff
@@ -0,0 +1,27 @@
+# Inputs
```

Comment:
This file can be removed because it is automatically generated from the viash config when the project is build.